### PR TITLE
Tweak README to make it more beginner-friendly

### PR DESCRIPTION
**Pull request by sprt**
As a beginner, it wasn't immediately obvious to me how exactly to
properly create and *configure* a TPU node, which resulted in a run-time
error that I couldn't interpret.

Likewise, it took me a second to understand the difference between the
two sections (single node vs pod).

-Aurelien

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![CircleCI](https://circleci.com/gh/pytorch/xla.svg?style=svg)](https://circleci.com/gh/pytorch/xla)
 
-# How to Run PyTorch with TPUs
+# How to Run PyTorch with single TPU nodes
```
Comment (dlibenzi) [inline README.md]: `single TPU nodes` -> `Single TPU Nodes`

Comment (dlibenzi) [reply README.md]: I'd also try to see if we can squeeze in something like the link below (unless we have it elsewhere), as when I first read this, I thought Single Core:

https://cloud.google.com/tpu/docs/system-architecture

**Comment by sprt** [comment]
On second thought, may I suggest we replace the distributed training section with just a link to [Training PyTorch models on Cloud TPU Pods](https://cloud.google.com/tpu/docs/tutorials/pytorch-pod) (which we already have in the first section)? I feel like there's a lot of duplicated work there.

File: README.md [hunk 2]
```diff
@@ -128,8 +135,7 @@ Training on pods can be broken down to largely 3 different steps:
 2. Let's say the command you ran to run a v3-8 was: `XLA_USE_BF16=1 python test/test_train_imagenet.py --fake_data`.
 * To distribute training as a conda environment process:
 ```
-(torch-xla-nightly)$ cd /usr/share/torch-xla-nightly/pytorch/xla
-(torch-xla-nightly)$ python -m torch_xla.distributed.xla_dist --tpu=$TPU_POD_NAME --conda-env=torch-xla-nightly --env=XLA_USE_BF16=1 -- python test/test_train_imagenet.py --fake_data
+(torch-xla-nightly)$ python -m torch_xla.distributed.xla_dist --tpu=$TPU_POD_NAME --conda-env=torch-xla-nightly --env=XLA_USE_BF16=1 -- python /usr/share/torch-xla-0.5/pytorch/xla/test/test_train_imagenet.py --fake_data
```
Comment (dlibenzi) [inline README.md]: I assume you have verified this change, as the previous one was "supposed" 😄 to be working.

Comment (jysohn23) [reply README.md]: Yep looks like this makes it consistent with the tutorials. Thanks!

**Pull request merged**